### PR TITLE
BREAKING CHANGE: return txreqid and raw message

### PR DESCRIPTION
**Pull request by jeanlarrieux**
Ticket: BG-68940

### CODE REVIEW

File: modules/bitgo/test/v2/unit/wallet.ts [hunk 1]
```diff
@@ -2473,7 +2473,7 @@ describe('V2 Wallet:', function () {
           typedData.version = SignTypedDataVersion.V4;
           it('should sign typed data V4', async function () {
             typedData.txRequestId = txRequestId;
-            const signTypedDataTssSpy = sinon.spy(tssEthWallet, 'signTypedDataTss' as any);
+            sinon.spy(tssEthWallet, 'signTypedDataTss' as any);
```
Comment (sachushaji) [inline modules/bitgo/test/v2/unit/wallet.ts:2476]: Is this used anymore in test?, If not kindly remove this

File: modules/bitgo/test/v2/unit/wallet.ts [hunk 2]
```diff
@@ -2483,15 +2483,12 @@ describe('V2 Wallet:', function () {
...
-            const actualArg = signTypedDataTssSpy.getCalls()[0].args[0];
-            const expected = '1901493fbf2ae1c27c3ced26a89070c6ab5d3fbf37ed778de9378e7703b7d1f116b3883077a61826129b98b622e54fc68c5008d1b1c16552e1eda6916f870d719220';
-            actualArg.typedData.typedDataEncoded.toString('hex').should.equal(expected);
+            signedTypedData.should.deepEqual(expected);
           });
 
           it('should sign typed data V4 when custodianMessageID is provided', async function () {
             typedData.txRequestId = txRequestId;
-            const signTypedDataTssSpy = sinon.spy(tssEthWallet, 'signTypedDataTss' as any);
+            sinon.spy(tssEthWallet, 'signTypedDataTss' as any);
```
Comment (sachushaji) [inline modules/bitgo/test/v2/unit/wallet.ts:2491]: Is this used anymore in test?, If not kindly remove this

File: modules/bitgo/test/v2/unit/wallet.ts (hunk 1 above)
Comment (jeanlarrieux) [reply modules/bitgo/test/v2/unit/wallet.ts:2476]: I think you are looking at a stale version of the PR ([latest version](https://github.com/BitGo/BitGoJS/blob/a90f86e57c27e4e66bbaee705bfddd0b4d1c6e21/modules/bitgo/test/v2/unit/wallet.ts#L2476))

File: modules/bitgo/test/v2/unit/wallet.ts (hunk 2 above)
Comment (jeanlarrieux) [reply modules/bitgo/test/v2/unit/wallet.ts:2491]: I think you are looking at a stale version of the PR ([latest version](https://github.com/BitGo/BitGoJS/blob/a90f86e57c27e4e66bbaee705bfddd0b4d1c6e21/modules/bitgo/test/v2/unit/wallet.ts#L2491))

**Comment by mmcshinsky-bitgo** [comment]
It looks like the breaking change commit still has the `bitgo` module in it.

**Comment by mmcshinsky-bitgo** [comment]
> > It looks like the breaking change commit still has the `bitgo` module in it.
> 
> Yes because tests are located in the `bitgo` module, so needed to make changes in the test as well.

You'll want to separate out the commits between them since there are no breaking changes for usage functionality in `bitgo`.

File: modules/sdk-core/src/bitgo/baseCoin/iBaseCoin.ts [hunk 3]
```diff
@@ -364,7 +364,11 @@ export type SignedTransaction =
   | FullySignedTransaction
   | SignedTransactionRequest;
 
-export type SignedMessage = SignedTransactionRequest | string;
+export interface SignedMessage {
+  txHash: string;
```
Comment (karankurbur) [inline modules/sdk-core/src/bitgo/baseCoin/iBaseCoin.ts:368]: change field name to `singedMessage` - txHash is not relevant for message signing

Comment (jeanlarrieux) [reply modules/sdk-core/src/bitgo/baseCoin/iBaseCoin.ts:368]: That may create issues as Front end is expecting this nomenclature.

**Review by karankurbur — DISMISSED** [review]

**Review by mmcshinsky-bitgo — APPROVED** [review]

**Pull request merged**